### ISSUE TITLE
[Bug] <Tsumino>: <Pages take too long to load>

### ISSUE THREAD

**Issue by GutHib1209** [tachiyomiorg/tachiyomi-extensions#6054]
## Device information
- Tachiyomi version: 0.10.9

## Source information
- Name: Tsumino
- Extension version: 1.2.4

## Steps to reproduce
1. Tap "Latest" or Tap the extension name
2. Tap on desired doujin

### Expected behavior
The pages for the chapter selected should load fairly quickly (5-10 seconds) with succesive pages preloading to facilitate a smooth read

### Actual behavior
Pages take a long time to load (30-60 seconds) with preloading occuring even slower

**Comment by scb261** [comment]
Loads pretty fast for me (1-2 seconds per page). Try [troubleshoot](https://tachiyomi.org/help/guides/troubleshooting-problems/#diagnosis). Maybe they're throttling your ip, how much did you download recently?

**Comment by yooman1213** [comment]
Apparently the web is undergoing some changes it doesn't just effect the app

The long loading time is due to the website problem, now the website already fix it by changing their image databases with .bin file so it loads faster 

But sadly broke the tachiyomi app to read it 

Because the downloaded file now in .bin file the app can't read it so it show as " no page found " even though the file already downloaded

**Comment by Seishirou101** [comment]
I don't believe that's the case. There was a similar issue and the op said it was because the root dir was in Chinese.  (Google translate) #4804.  I also verified that your issue #4794, regardless if I download on tachi or sy, I get jpgs like arkon does in #4804
